tasks/copy_grsecurity_files.yml

```diff
@@ -0,0 +1,11 @@
+---
+# Not using get_url because it doesn't follow redirects.
+- name: Copy grsecurity patch.
+  copy:
```

Comment:
I would have preferred a copy with a `with_items` loop but this is not a blocker for merge.

Reply:
Yes, that would have been cleaner.